maps/work_default.ahk

```diff
@@ -75,11 +76,7 @@ Return
 Return
 
 
-; WORD ------------------------------------------------------------------------
-#IfWinActive ahk_class OpusApp
-   PrintScreen::AppsKey
+; WORD/OUTLOOK ----------------------------------------------------------------
+#If WinActive("ahk_class OpusApp") || WinActive("ahk_class rctrl_renwnd32")
+  PrintScreen::AppsKey
 Return
-
-
-; SYMBOL KEYS -----------------------------------------------------------------
->+s::send §
```

Comment:
This was causing an issue and it's not used frequently anyways; going to remove for now.